rust-code-analysis-cli/src/main.rs

```diff
@@ -78,7 +78,7 @@ fn act_on_file(language: Option<LANG>, path: PathBuf, cfg: &Config) -> std::io::
         };
         action::<Dump>(&language, source, &path, pr, cfg)
     } else if cfg.metrics {
-        let cfg = MetricsCfg {
+        let cfg = SpacesCfg {
```

Comment:
I don't understand why you changed the name here, maybe a `sed s//g` issue

Reply:
Because I renamed `MetricsCfg` into `SpacesCfg` since it's a configuration for functions spaces.